styles.css

```diff
@@ -288,7 +288,12 @@ input[type='checkbox'] + label {
 }
 
 .popup-content address {
+  padding-bottom: 5px;
+}
+
+.popup-content .updated-at {
   padding-bottom: 20px;
+  font-style: italic;
```

Comment:
Italic is a bit arbitrary here, we now have two sections that are italic on top of each other:

![Screen Shot 2020-06-04 at 10 12 36 PM](https://user-images.githubusercontent.com/108094/83839305-8a1f8d80-a6b0-11ea-8e4d-02ce9a7270bc.png)

Reply:
Good point. I like last updated as italic and moving the address to no italic since the blue and underline call it out enough